crates/next-custom-transforms/src/transforms/page_static_info/collect_exports_visitor.rs

```diff
@@ -79,7 +80,11 @@ impl Visit for CollectExportsVisitor {
                         export_info.generate_image_metadata =
                             Some(name.sym == "generateImageMetadata");
                         export_info.generate_sitemaps = Some(name.sym == "generateSitemaps");
-                        export_info.generate_static_params = name.sym == "generateStaticParams";
+                        export_info.generate_static_params = if name.sym == "generateStaticParams" {
+                            Some(name.span())
+                        } else {
+                            None
+                        };
```

Comment:
You could use `(name.sym == "generateStaticParams").then_some(name.span())` or `(name.sym == "generateStaticParams").then(|| name.span())`